npm-shrinkwrap.json

```diff
@@ -18527,6 +18539,11 @@
         }
       }
     },
+    "vscode-languageserver-types": {
```

Comment:
`vscode-languageserver-types` as a production dependency? Seems to be a dependency of the newly added `dockerfile-ast` module. Only 300KB though so never mind, we can't fix the world... :-)